app/src/main/java/uk/ac/ebi/atlas/home/JsonExperimentsSummaryController.java

```diff
@@ -17,12 +17,7 @@
 
 import static com.google.common.collect.ImmutableList.toImmutableList;
 import static uk.ac.ebi.atlas.model.card.CardIconType.IMAGE;
-import static uk.ac.ebi.atlas.model.experiment.ExperimentType.MICROARRAY_1COLOUR_MICRORNA_DIFFERENTIAL;
-import static uk.ac.ebi.atlas.model.experiment.ExperimentType.MICROARRAY_1COLOUR_MRNA_DIFFERENTIAL;
-import static uk.ac.ebi.atlas.model.experiment.ExperimentType.MICROARRAY_2COLOUR_MRNA_DIFFERENTIAL;
-import static uk.ac.ebi.atlas.model.experiment.ExperimentType.PROTEOMICS_BASELINE;
-import static uk.ac.ebi.atlas.model.experiment.ExperimentType.RNASEQ_MRNA_BASELINE;
-import static uk.ac.ebi.atlas.model.experiment.ExperimentType.RNASEQ_MRNA_DIFFERENTIAL;
+import static uk.ac.ebi.atlas.model.experiment.ExperimentType.*;
```

Comment:
We prefer not to use `*` so that only necessary items are imported. Can you please revert it and add the only ones required in this class?

Reply:
This was all smarty intelliJ, are you sure?

Reply:
or is there a setting to turn that off?

Reply:
Any way, not a problem to revert.

Reply:
Addressed.